results/client/client.go

```diff
@@ -0,0 +1,73 @@
+package client
+
+import (
+	"context"
+	"github.com/google/uuid"
+	proto "github.com/macformula/hil/results/client/generated"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const addr = "localhost:31763"
+
+type ResultsClient struct {
+	conn   *grpc.ClientConn
+	client proto.TagTunnelClient
+}
+
+func (r *ResultsClient) Open(ctx context.Context) error {
+	conn, err := grpc.DialContext(ctx, addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		return err
```

Comment:
for all error handling, user `errors.Wrap`. Put the name of the function that was called in lower case. I'll put up a couple examples.
```suggestion
		return errors.Wrap(err, "dial context")
```